### PR TITLE
Adding support for Feather ThinkInk

### PR DESCRIPTION
**Pull request by BlitzCityDIY**
Adding support for the Feather ThinkInk. Tested with u2if firmware

### CODE REVIEW

**Comment by makermelissa** [comment]
I added @caternuson as a reviewer since he originally added all the U2IF boards.

**Review by makermelissa — APPROVED** [review]
It looks good to me at a glance. I'll leave unmerged for now so Carter can take a look as well unless you're good with this.

File: src/adafruit_blinka/board/feather_epd_u2if.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+D10 = pin.GP10
+D11 = pin.GP11
+D12 = pin.GP12
+D13 = pin.GP13
+D24 = pin.GP24
+D25 = pin.GP25
+
+A0 = pin.GP26
+A1 = pin.GP27
+A2 = pin.GP28
```
Comment (caternuson) [inline src/adafruit_blinka/board/feather_epd_u2if.py:38]: Need A3 also?

**Comment by caternuson** [comment]
Looks like this is the associated Platform Detect PR, adding for general ref:
https://github.com/adafruit/Adafruit_Python_PlatformDetect/pull/286

@BlitzCityDIY Did not test (do not have hardware), but looks generally OK. One missing pin define for A3. What actual u2if firmware was used for testing?

**Comment by BlitzCityDIY** [comment]
thanks @caternuson - just added A3. i have a fork here for u2if (haven't PR'd yet): https://github.com/BlitzCityDIY/u2if

**Comment by caternuson** [comment]
oh cool - you built one locally. awesome. lgtm.

**Review by caternuson — APPROVED** [review]

**Pull request merged**